src/main/kotlin/com/sourcegraph/cody/auth/ui/LoadingAccountsDetailsProvider.kt

```diff
@@ -41,9 +42,13 @@ abstract class LoadingAccountsDetailsProvider<in A : Account, D : AccountDetails
           }
           .exceptionally {
             val error = CompletableFutureUtil.extractError(it)
-            val errorMessage =
-                error.localizedMessage.takeWhile { c -> c.isLetterOrDigit() || c.isWhitespace() }
-            DetailsLoadingResult(null, null, errorMessage, false)
+            if (error.message == UNAUTHORIZED_ERROR_MESSAGE) {
+              DetailsLoadingResult(null, null, error.localizedMessage, true)
```

Comment:
<img width="992" alt="image" src="https://github.com/sourcegraph/jetbrains/assets/19799111/274d928b-4af2-4a3f-87b0-eb4276c3b75c">

When the user is unauthorized there is an action to login 🚀